intellij-toml/src/test/kotlin/org/toml/TomlTestBase.kt

```diff
@@ -0,0 +1,28 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.toml
+
+import com.intellij.testFramework.fixtures.BasePlatformTestCase
+import org.intellij.lang.annotations.Language
+
+abstract class TomlTestBase: BasePlatformTestCase() {
+    @Suppress("TestFunctionName")
+    protected fun InlineFile(@Language("TOML") text: String, name: String = "Cargo.toml") {
```

Comment:
```suggestion
    protected fun InlineFile(@Language("TOML") text: String, name: String = "example.toml") {
```

Just non to mislead anyone that these tests somehow related to Rust and Cargo